student.rb

```diff
@@ -9,8 +9,11 @@ def initialize(age, name, classroom, parent_permission: true)
   end
 
   def play_hooky
-    # rubocop:disable RuleByName
     "¯\(ツ)/¯"
-    # rubocop:disable RuleByName
+  end
+
+  def classrooms(classroom)
+    @classroom = classroom
+    classroom.student.push(self) unless classroom.student.include?(self)
   end
```

Comment:
- [ ] As requested by the previous reviewer, I am still getting an error when I try to create a student. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/201352205-abf4588a-1466-4803-8040-19647f95f9ba.png)

- [ ] I would advise to remove the classroom parameter  when entering information for the student as it is not part of the requirements in the [Video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), but it should ask to apply parent permission